.vscodeignore

```diff
@@ -15,3 +15,10 @@ sonar-project.properties
 test Fixture with speci@l chars
 test-resources
 .test-extensions
+camel-ls-sbom.json
+docs
+coverage
+.camel-jbang
+node_modules
```

Comment:
to verify: i think that node_modules is mandatory in the vsix file

Reply:
This was inspired from LSP coverage PR - https://github.com/camel-tooling/camel-lsp-client-vscode/pull/1755/files#diff-0a721cd4753ff50bbbe1237397c3c7692080254fa766268fcc8b05ef633c50bfR28

Reply:
this is what i remembered, need to reinclude it and do another release: https://github.com/camel-tooling/camel-lsp-client-vscode/commit/f9cd25dff7ed457efa1a0a0255b07b0003824436

Reply:
ok, my bad - I overlooked this commit, will remove